tests/select_file/BUILD

```diff
@@ -0,0 +1,34 @@
+load("//rules:select_file.bzl", "select_file")
+load("//rules:diff_test.bzl", "diff_test")
+
+filegroup(
+    name = "fg",
+    srcs = [
+        "subdir/inner.txt",
+        ":select_me.txt",
+    ],
+)
+
+select_file(
+    name = "select_me",
+    srcs = ":fg",
+    subpath = "select_me.txt",
+)
+
+select_file(
+    name = "select_inner",
+    srcs = ":fg",
+    subpath = "subdir/inner.txt",
+)
+
+diff_test(
```

Comment:
add some unique content to the two test files? a `diff_test` between `subdir/inner.txt` and `select_me.txt` will pass, since they're both empty.